larray/random.py

```diff
@@ -379,33 +379,34 @@ def choice(choices=None, axes=None, replace=True, p=None, meta=None):
         Otherwise, if the resulting axes have a shape of, e.g., ``(m, n, k)``, then ``m * n * k`` samples are drawn.
     replace : boolean, optional
         Whether the sample is with or without replacement.
-    p : 1-D array-like, optional
+    p : array-like, optional
         The probabilities associated with each entry in choices.
-        If p is a 1-D LArray, choices are taken from its axis.
+        If p is an LArray, choices are taken from its axes. If p is N dimensional, each cell represents the probability
+        that the combination of labels will occur.
         If not given the sample assumes a uniform distribution over all entries in choices.
```

Comment:
I wonder if it wouldn't be better to say "If p is a 1-D LArray, choices are taken from its axis. If p is N dimensional, each cell represents the probability that the combination of labels will occur."
I don't know.

Reply:
I hesitated with that too. It seemed redundant, but is probably easier to understand, so ok, I'll change that.